documentation/src/main/asciidoc/topics/proc_configuring_cache_tracing.adoc

```diff
@@ -0,0 +1,71 @@
+[id='tracing_cache']
+= Configure tracing at cache level
+
+Once the tracing is link:#configuring-tracing_{context}[configured at server level],
+it will be automatically enabled by default for all caches.
+A cache configuration level of tracing allows on the other hand to enable or disable it at cache level and at runtime.
+
+[discrete]
+== Tracing categories
+
+Several categories are potentially traced:
+
+* Container. That are all the main cache operations, such as replace, put, clear, getForReplace, remove operations and size. With the exception of all the getting operation.
+* Cluster. All the operations involved at clustering level, e.g., to align a replicas.
+* X-Site. All the operations involved at cross-site level, e.g, to align a remote site.
+* Persistence. All the operations involving persistence (cache store).
+
+Each of category can be enabled/disabled at start time or runtime using the related tracing configuration attributes.
+By default only the container category is enabled.
+
+There is also the `Security` category, to trace security audit events.
+This category is configured globally, not only at cache level, since their events can have different scopes (cache, container or server),
+non only cache scope.
+
+[discrete]
+== Enable/disable tracing for a given cache
+{brandname} applies the tracing configuration globally to all caches.
```

Comment:
Have to admit I don't understand what this is trying to say. Some tracing configuration elements are global and some are per cache. This section seems to be only about the cache, no?

Reply:
You're right. This sentence is definitively not clear. I'm rephrasing it. Thank you @wburns